test/conformance/api/v1alpha1/blue_green_test.go

```diff
@@ -67,7 +67,7 @@ func TestBlueGreenRoute(t *testing.T) {
 	// Setup Initial Service
 	t.Log("Creating a new Service in runLatest")
 	objects, _, err := v1a1test.CreateRunLatestServiceReady(t, clients, &names,
-		false /* https TODO(taragu) turn this on after helloworld test running with https */)
```

Comment:
I'm confused. You removed all these TODOs, but kept the value `false`, contradicting the explanation. What am I missing?